solver/src/settlement.rs

```diff
@@ -205,98 +258,117 @@ mod tests {
     }
 
     #[test]
+    #[allow(clippy::just_underscores_and_digits)]
     fn test_buy_order_surplus() {
         // Two goods are worth the same (100 each). If we were willing to pay up to 60 to receive 50,
         // but ended paying the price (1) we have a surplus of 10 sell units, so a total surplus of 1000.
+
+        // I hope there is a better way :)
```

Comment:
You could define yourself a macro or a small helper function.

Reply:
Done in a2a4e.